internal/slice/slice_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slice
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestRemoveString(t *testing.T) {
```

Comment:
if we keep the `slice` package would be good to have a test for the other utility as well

Reply:
ack